src/connection/coniql.ts

```diff
@@ -261,8 +261,15 @@ export class ConiqlPlugin implements Connection {
     this.wsClient.onDisconnected((): void => {
       log.error("Websockect client disconnected.");
       for (const pvName of Object.keys(this.subscriptions)) {
-        this.subscriptions[pvName].unsubscribe();
```

Comment:
maybe we could just replace this.subscriptions[pvName].unsubscribe() with this.subscriptions[pvName]?.unsubscribe();
Your fix seems fine though and probably nice to provide some logging like you have

Reply:
I agree that would be nicer, but I think if we aren't going to solve the underlying problem we should log it.